lib/core/macros.nim

```diff
@@ -444,7 +444,7 @@ proc ident*(name: string): NimNode {.magic: "StrToIdent", noSideEffect.}
 type
   BindSymRule* = enum    ## specifies how `bindSym` behaves
     brClosed,            ## only the symbols in current scope are bound
-    brOpen,              ## open wrt overloaded symbols, but may be a single
+    brOpen,              ## open regrading overloaded symbols, but may be a single
```

Comment:
Can someone clarify it a bit? I don't quite understand "open wrt overloaded symbols"? Maybe I'm missing something.

Reply:
A suitable substitution might be _in the case of_ or _for_.  The idea we want to communicate is that the result will not be open in the case of an unambiguous binding, while the result is always open in the subsequent enum value.

Reply:
thanks